analyzers/src/SonarAnalyzer.Common/Helpers/UnitTestHelper.cs

```diff
@@ -24,81 +24,47 @@
 
 // Note: useful comparison of the differing syntax across unit test frameworks
 // at https://xunit.github.io/docs/comparisons
-
```

Comment:
The link above doesn't work. If you can find replacement in 2 minutes, we can update it (to a single line version). Otherwise we don't need the comment.